src/go/k8s/pkg/resources/configmap.go

```diff
@@ -40,12 +40,14 @@ const (
 
 	cloudStorageCacheDirectory = "cloud_storage_cache_directory"
 
-	tlsDir   = "/etc/tls/certs"
-	tlsDirCA = "/etc/tls/certs/ca"
-
-	tlsAdminDir          = "/etc/tls/certs/admin"
-	tlsPandaproxyDir     = "/etc/tls/certs/pandaproxy"
-	tlsSchemaRegistryDir = "/etc/tls/certs/schema-registry"
+	tlsRedpandaDir         = "/etc/tls/certs"
```

Comment:
nit: it would be a bit better to have these called tlsKafkaApiDir etc.

Reply:
✅